plugins/Satellites/src/Satellite.cpp

```diff
@@ -864,6 +865,8 @@ SatFlags Satellite::getFlags() const
 		flags |= SatHEO;
 	if (eccentricity < 0.25 && (inclination>=25. && inclination<=180.) && (orbitalPeriod>=1100. && orbitalPeriod<=2000.))
 		flags |= SatHGSO;
+	if (StelApp::getInstance().getCore()->getJD() - tleEpochJD > 30.) // TLE epoch is over 30 days old
```

Comment:
Not sure: is current simulation time critical, or rather real-world time? (getJDfromSystem() ?) And: are TLE also outdated if I go 4 months back? in the latter case, at least test abs(JD-tleEpochJD)

Reply:
Simulation time is critical IMHO. TLE is valid on time range 72 hours (probably up to 100 hours with low accuracy). I'm use multiplicator 10 for the 72 hours (3 days * 10 = 30 days), but yes, `abs()` will be good here.